src/lwc-dev-server/index.ts

```diff
@@ -72,6 +74,7 @@ async function createLWCServerConfig(rootDir: string, logger: Logger): Promise<S
     paths: namespacePaths,
     workspace: await LwcDevServerUtils.getLocalDevServerWorkspace(),
     targets: ['LEX'], // should this be something else?
+    identityToken,
```

Comment:
in order to be aligned with the rest of the way we pass the params, maybe we should update this to `identityToken: identityToken" instead?